requirements/edx/local.txt

```diff
@@ -1,12 +1,13 @@
 # Python libraries to install that are local to the edx-platform repo
 -e .
--e common/lib/calc
+
 -e common/lib/capa
--e common/lib/chem
--e common/lib/dogstats
--e common/lib/safe_lxml
--e common/lib/sandbox-packages
--e common/lib/symmath
 -e common/lib/xmodule
 
+-e openedx/core/lib/calc
```

Comment:
Do we have clearly defined best practices around imports somewhere?

As an example, should:
```
import calc
```
be switched to:
```
import openedx.core.lib.calc
```
?

Or is this wrong?